infrasim/model/elements/storage.py

```diff
@@ -77,3 +76,17 @@ def handle_parms(self):
         # controller attributes if there are some
         # common attributes for all controllers
         # add them into self._attributes here.
+
+    def get_drvs_arg_option(self):
+        if len(self._drive_list) == 0:
+            return None
+        if self._controller_info.get('connectors', None) is None:
+            return None
+        # handle drive options
+        for drive_obj in self._drive_list:
+            drive_obj.handle_parms()
+        drv_args = []
+        for drive_obj in self._drive_list[:]:
```

Comment:
Good practice, the `[:]` makes a slice copy of this list, which will not impact the original list.